apps/hash-graph/libs/graph/src/store/postgres/query/table.rs

```diff
@@ -399,332 +540,151 @@ fn transpile_json_field(
-impl OntologyIds {
-    fn transpile_column(self, table: &impl Transpile, fmt: &mut fmt::Formatter) -> fmt::Result {
-        let column = match self {
-            Self::OntologyId => "ontology_id",
-            Self::BaseUrl => "base_url",
-            Self::Version => "version",
-            Self::LatestVersion => "latest_version",
-        };
-        table.transpile(fmt)?;
-        write!(fmt, r#"."{column}""#)
-    }
-
-    pub const fn parameter_type(self) -> ParameterType {
-        match self {
-            Self::OntologyId => ParameterType::Uuid,
-            Self::BaseUrl => ParameterType::Text,
-            Self::Version | Self::LatestVersion => ParameterType::OntologyTypeVersion,
-        }
-    }
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
+pub enum OwnedOntologyMetadata {
+    OntologyId,
+    WebId,
 }
 
-impl OntologyOwnedMetadata {
-    fn transpile_column(self, table: &impl Transpile, fmt: &mut fmt::Formatter) -> fmt::Result {
-        let column = match self {
-            Self::OntologyId => "ontology_id",
-            Self::WebId => "web_id",
-        };
-        table.transpile(fmt)?;
-        write!(fmt, r#"."{column}""#)
-    }
-
-    pub const fn parameter_type(self) -> ParameterType {
+impl DatabaseColumn for OwnedOntologyMetadata {
+    fn parameter_type(self) -> ParameterType {
         match self {
             Self::OntologyId | Self::WebId => ParameterType::Uuid,
         }
     }
-}
 
-impl OntologyExternalMetadata {
-    fn transpile_column(self, table: &impl Transpile, fmt: &mut fmt::Formatter) -> fmt::Result {
-        let column = match self {
-            Self::OntologyId => "ontology_id",
-            Self::FetchedAt => "fetched_at",
-        };
-        table.transpile(fmt)?;
-        write!(fmt, r#"."{column}""#)
+    fn nullable(self) -> bool {
+        false
     }
 
-    pub const fn parameter_type(self) -> ParameterType {
+    fn as_str(self) -> &'static str {
         match self {
-            Self::OntologyId => ParameterType::Uuid,
-            Self::FetchedAt => ParameterType::Timestamp,
+            Self::OntologyId => "ontology_id",
+            Self::WebId => "web_id",
         }
     }
 }
 
-impl OntologyAdditionalMetadata {
-    fn transpile_column(self, table: &impl Transpile, fmt: &mut fmt::Formatter) -> fmt::Result {
-        let column = match self {
-            Self::OntologyId => "ontology_id",
-            Self::AdditionalMetadata => "additional_metadata",
-        };
-        table.transpile(fmt)?;
-        write!(fmt, r#"."{column}""#)
-    }
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
+pub enum DataTypes {
+    OntologyId,
+    Schema,
+}
 
-    pub const fn parameter_type(self) -> ParameterType {
+impl DatabaseColumn for DataTypes {
+    fn parameter_type(self) -> ParameterType {
         match self {
             Self::OntologyId => ParameterType::Uuid,
-            Self::AdditionalMetadata => ParameterType::Object,
+            Self::Schema => ParameterType::Any,
```

Comment:
why the change from `Object` -> `Any`?

Reply:
It didn't change, `git` just didn't calculate the diff propertly